plugin.php

```diff
@@ -12,12 +13,13 @@ function get_secure_menu_entries($page, $user)
         {
             $ret['Ticket System Data Entry']=$page->secure_root.'tickets/_admin/data.php';
         }
+        
         return $ret;
     }
 
     function get_plugin_entry_point()
     {
-        return array('name'=>'Ticket Registration/Transfer', 'link'=>'tickets/index.php');
```

Comment:
if we decide that's what we want then lets create a separate ticket to track that change.. 
it's not directly relevant to softening links